conda/meta.yaml

```diff
@@ -72,3 +74,5 @@ requirements:
     - ts-xml
     - ts-guitool
     - ts-tcpip
+    - ts-salobj {{ salobj_version }}
```

Comment:
This should be only ts-salobj.

Reply:
Fixed in the commit: [96a18af](https://github.com/lsst-ts/ts_hexgui/pull/4/commits/96a18af29ed0b8ab30ae6384ee503af88bc19441). Thanks!